sdcm/nemesis.py

```diff
@@ -4413,6 +4419,90 @@ def disrupt_add_remove_mv(self):
         finally:
             self.unset_current_running_nemesis(cql_query_executor_node)
 
+    def disrupt_toggle_audit(self):
+        """
+            Enable audit log with all categories and user keyspaces (if audit already enabled, disable it and finish the Nemesis),
+            verify audit log content,
+            reduce categories by excluding DML and QUERY,
+            verify DDL are logged in audit log correctly. Leaves audit log enabled this way.
+        """
+        if not self.target_node.is_enterprise:
+            raise UnsupportedNemesis("Auditing feature is only supported by Scylla Enterprise")
+
+        if not (test_ks_cf_list := self.cluster.get_non_system_ks_cf_list(db_node=self.target_node)):
+            raise UnsupportedNemesis("Not found non-system keyspaces")
+
+        audit = Audit(self.cluster)
+
+        if audit.is_enabled():
+            InfoEvent("Audit was enabled -> disabling it").publish()
+            audit.disable()
+            return
```

Comment:
Maybe we need to set this nemesis as "skipped" and not "succeeded". And reason "Audit was enabled -> disabling it. Nemesis were not run". Something like this

Reply:
I think this is a good idea